tools/src/CSVFileImport.cc

```diff
@@ -311,7 +311,7 @@ int main(int argc, char* argv[]) {
   int opt;
   char *tail;
   do {
-    opt = getopt_long(argc, argv, "i:o:s:b:c:p:t:h", longOptions, ORC_NULLPTR);
+    opt = getopt_long(argc, argv, "i:o:d:s:c:b:t:h", longOptions, ORC_NULLPTR);
```

Comment:
Can the 'i:' and 'o:' in the third param of the getopt_long function be deleted together?

Reply:
Sure